examples/hello-world/job_api/tf/requirements.txt

```diff
@@ -1,2 +1,2 @@
 nvflare~=2.5.0rc
-tensorflow==2.15.0
+tensorflow[and-cuda]
```

Comment:
Do we want this flag here?

What happen if users has some Cuda already installed?

Reply:
This seems to be the recommended way to install tensorflow https://www.tensorflow.org/install/pip

Reply:
got it, are we planning on adding readme files for these examples in job_api?

Reply:
Yes, those will come.